src/ui/foundation/QueryClientProvider/QueryClientProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import {
+  QueryClientProvider as TanstackQueryClientProvider,
+  QueryClient,
+} from '@tanstack/react-query';
+import React from 'react';
+
+const queryClient = new QueryClient();
```

Comment:
Not sure if that should be chilling here or in a state instead?

I did this according to the docs.